src/app/Main/Body/QueryPage/Parameter/QueryPage-ParameterContent.component.tsx

```diff
@@ -20,12 +20,12 @@ export function QueryPageParameterContentComponent(): JSX.Element {
     const { filter } = useContext(FilterContext);
     const { t } = useTranslation(["QueryPage"]);
 
-    const displayFilter = _.cloneDeep(filter);
+    const displayFilter = _.cloneDeep(filter.selectedFilter);
     mainFilterAttributes.forEach((element) => {
-        if (filter[element].length === 0) {
+        if (!_.isEmpty(filter.selectedFilter) || filter.selectedFilter[element].length === 0) {
```

Comment:
Reject: i think you want to check if filter.selected isEmpty and not !isEmpty
And actually i think you want to check that filter.selectedFilter[element] === undefined or filter.selectedFilter[element].length === 0

Reply:
!_.isEmpty => _.isEmpty was done with isolate api ticket

Reply:
and the second part? This is the same issue as stated above. If your filter.selectedFilter object is allowed to be empty, checking for non-emptiness does not imply that the filter.selectedFilter[element] is set. So the app would crash anyway. But i think you refactored your code already so that the default of selectedFilter is set to an appropriate object with empty arrays, so you can remove the isEmpty check safely. Or am i wrong?

Reply:
You are right. Done. 
- Remove useless _.isEmpty() condition